src/pybind/ceph_volume_client.py

```diff
@@ -82,6 +82,13 @@ def __str__(self):
         return "Error {0} (\"{1}\") while {2}".format(
             self._result_code, self._result_str, self._action)
 
+class TooManyVersionMismatch(Exception):
```

Comment:
nit: another option could be to reuse `rados.ObjectBusy` (or maybe subclass it to be specific for this exception).

@rishabh-d-dave @ajarr @batrick ?

Reply:
If we keep iterations, IMO, this too would be nice.